roles/yoda_rulesets/tasks/yoda-ruleset.yml

```diff
@@ -320,6 +320,15 @@
   when: (not ansible_check_mode) and (formats_installed.stderr.find('does not exist') >= 0)
 
 
+- name: Ensure admin has permissions to make changes to file formats
+  become_user: "{{ irods_service_account }}"
+  become: true
+  # noqa no-changed-when
+  ansible.builtin.command:
+    ichmod -rM own priv-admin /{{ irods_zone }}/yoda/file_formats
+  when: priv_groups.stdout.find('priv-admin') != -1
```

Comment:
We can add `not ansible_check_mode` to prevent it from running in check mode.

Reply:
added check